src/gui/guiEngine.cpp

```diff
@@ -105,15 +105,33 @@ void MenuMusicFetcher::fetchSounds(const std::string &name,
 		return;
 	m_fetched.insert(name);
 	std::string base;
-	base = porting::path_share + DIR_DELIM + "sounds";
-	dst_paths.insert(base + DIR_DELIM + name + ".ogg");
+	std::vector<fs::DirListNode> list;
+	base = porting::path_share + DIR_DELIM;
+	// Add standard main menu sounds
+	dst_paths.insert(base + "sounds" + DIR_DELIM + name + ".ogg");
 	int i;
 	for(i=0; i<10; i++)
-		dst_paths.insert(base + DIR_DELIM + name + "."+itos(i)+".ogg");
-	base = porting::path_user + DIR_DELIM + "sounds";
-	dst_paths.insert(base + DIR_DELIM + name + ".ogg");
-	for(i=0; i<10; i++)
-		dst_paths.insert(base + DIR_DELIM + name + "."+itos(i)+".ogg");
+		dst_paths.insert(base + "sounds" + DIR_DELIM + name + "."+itos(i)+".ogg");
+	// Add game specific main menu sounds
```

Comment:
While you're already at it; could you please move this to a local function, such as
```
auto add_paths = [base, dst_paths, name] (const std::string &loc) {
	dst_paths.insert(base + loc + DIR_DELIM + name + ".ogg");
	for (int i = 0; i < 10; i++)
		dst_paths.insert(base + loc + DIR_DELIM + name +"." + itos(i) + ".ogg");
};

add_paths("sounds");
add_paths("games" + DIR_DELIM + name);
add_paths("games" + DIR_DELIM + name + DIR_DELIM + "menu");
```
or something like that?